src/gluonts/dataset/loader.py

```diff
@@ -101,16 +57,43 @@ class DataLoader(Iterable[DataEntry]):
     def __init__(
         self,
         dataset: Dataset,
+        *,
```

Comment:
What is this?

Reply:
Enforces all parameters afterwards to be keyword-only.